src/components/Offers/Offers.tsx

```diff
@@ -1,46 +1,69 @@
-﻿import React, { ChangeEvent, useState } from 'react';
+﻿import React, { ChangeEvent, FocusEvent, useState } from 'react';
 import OffersList from '../OffersList/OffersList';
-import { Offer, SearchTerm, SearchVariant } from '../../types/types';
-import { useOffers } from '../../hooks/useOffers';
-import { highlightText } from '../../utils/highlightText';
-import { useOutsideClick } from '../../hooks/useOutsideClick';
 import SearchInput from '../SearchInput/SearchInput';
 import SearchWrapper from '../SearchWrapper/SearchWrapper';
+import SuggestionsList from '../SuggestionsList/SuggestionsList';
+import ResultsInfo from '../ResultsInfo/ResultsInfo';
+import { useOffers } from '../../hooks/useOffers';
```

Comment:
**nitpick:** `src/hooks` is mostly for UI related reusable hooks. These offers are data related hooks. It should be kept somewhere `src/data/offer/useOffers`. Keep it in mind the same way like in eDashboard